tools/githooks/lib/src/messages.dart

```diff
@@ -10,6 +10,9 @@ const String _reset = '\x1B[0m';
 /// Prints a reminder to stdout to run `gclient sync -D`. Uses colors when
 /// stdout supports ANSI escape codes.
 void printGclientSyncReminder(String command) {
+  if (command == 'pre-rebase') {
```

Comment:
It would be cleaner to just remove the call to this function from https://github.com/flutter/engine/blob/main/tools/githooks/lib/src/pre_rebase_command.dart#L19, I think.

Reply:
Ah yes that would be better, let me fix it.  Thanks @zanderso